core/gui/src/index.html

```diff
@@ -43,6 +43,13 @@
     <link
       href="https://fonts.googleapis.com/icon?family=Material+Icons&display=block"
       rel="stylesheet" />
+
+    <!-- Add JSON Editor CSS and JS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/jsoneditor@latest/dist/jsoneditor.min.css"
```

Comment:
Ditto.

Reply:
Using a CDN to include CSS files is a common practice. The URL is from jsdelivr. And Jsdeliver is very common used, and generally reliable.